src/utils/isClient.ts

```diff
@@ -0,0 +1,8 @@
+// A utility to check if the code is running on the client (vs. nextjs server)
+const isClient = !!(
```

Comment:
Thank you for adding this, it was bothering me having to copy and paste `isClient` multiple times haha

Reply:
yeah, same haha.